deployments/migrations/0042_auto_20211117_1527.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.24 on 2021-11-17 15:27
```

Comment:
This migration file seems to be removable. Does the same as 0041_auto_20211110_0924.py, except 'IFRC + NS' is 'IFRC_NS', which can be changed manually in 0041_auto_20211110_0924.py.